deploy/crds/cloudingress.managed.openshift.io_apischemes_crd.yaml

```diff
@@ -1,38 +1,67 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: apischemes.cloudingress.managed.openshift.io
+  name: APISchemes.cloudingress.managed.openshift.io
```

Comment:
purely out of curiosity, does case matter here? is it beneficial to have it capitalize vs not? (this is for my understanding, cause I don't know

Reply:
Primarily so VSCode would stop complaining. The Go linter didn't really like it lowercase. It happens to carry across, I suppose?